app/_src/guides/otel-metrics.md

```diff
@@ -144,9 +145,26 @@ spec:
     backends:
       - type: OpenTelemetry
         openTelemetry:
-          endpoint: opentelemetry-collector.mesh-observability.svc:4317
-' | kubectl apply -f -
+          endpoint: opentelemetry-collector.mesh-observability.svc:4317' | kubectl apply -f -
 ```
+{% endif_version %}
+{% if_version gte:2.9.x %}
```

Comment:
I think we don't need the two if_versions, we can fix it for all versions right?

Reply:
The policy is slightly different in 2.9.x, we are using syntactic sugar to omit topLevel targetRef

Reply:
Ah sorry ok I thought it was the newline at the beginning and end